### DISCUSSION TITLE
Refresh Interval

### DISCUSSION THREAD

**Discussion by yurijserrano** [mdelobelle/metadatamenu#137] [Q&A] [answered]
Good Morning 

I would like to know which value are you using in the Dataview plugin, more specifically for the **Refresh Interval** option to have the value update almost instantly as can be seen in the video right below:

**[VIDEO](
https://youtu.be/7bvIAkJf0OE?t=532)**

I am waiting to hear from you

**Comment by mdelobelle** [comment, marked as answer]
Hi, the refresh interval is 250ms, I have a vault with 4000+ files

**Reply by yurijserrano** [reply]
Thank you so much for the info.

I am testing some values to have the best experience.

When I finish my tests, I will include a detailed report about this.


Have a wonderful day.
